src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = element.querySelector('.js-Slideshow-slides'),
+    slides = element.querySelectorAll('.js-Slideshow-slide'),
+    nextTrigger = element.querySelector('.js-Slideshow-next'),
+    prevTrigger = element.querySelector('.js-Slideshow-prev'),
+    currentCountElement = element.querySelector('.js-Slideshow-current'),
+    totalCountElement = element.querySelector('.js-Slideshow-total'),
```

Comment:
@erikjung Also see [this comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/350/files#r72335530).